source/configuration_manual/authentication/caching.rst

```diff
@@ -73,16 +73,14 @@ With other databases Dovecot doesn't know what could affect caching, so
 you have to tell Dovecot manually. The following databases require
 specifying the cache key:
 
--  pam
-
 -  bsdauth
 
 For example if the PAM lookup depends on username and service, you can
 use:
 
 ::
 
-   passdb pam {
+   passdb bsdauth {
```

Comment:
I just changed this to bsdauth so when we config-rewrite it it can be either removed or updated then

Reply:
OK, but change also the "For example if the PAM lookup" above it.

Reply:
and should be separate commit imo

Reply:
Made it separate now